docs/contribute/index.md

```diff
@@ -2,7 +2,7 @@
 description: Documentation contribution guide
 ---
 
-# Contribute to the documentation
+# Contribute to the Doctools documentation
```

Comment:
Are we referring to all the doc sites as "Doctools documentation"?

Reply:
No, you're right

Reply:
```suggestion
# Contribute to the documentation
```